types/node/fs/promises.d.ts

```diff
@@ -29,7 +29,12 @@ declare module 'fs/promises' {
         OpenMode,
         Mode,
         WatchOptions,
+        WatchEventType,
     } from 'node:fs';
+    interface FileChangeInfo<T = string | Buffer> {
```

Comment:
We should change this to `FileChangeInfo<T extends (string | Buffer)>`

Reply:
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/55033/commits/5542877d670cf9a493899b5dca25573ff545c703